src/index.html

```diff
@@ -11,6 +11,20 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class="form">
+      <div data-qa="big" class="form__search form__search--big">
+        <div class="form___icon form__icon--big"></div>
+        <input class="form__input form__input--big"
+          placeholder="Try “Los Angeles”"
+          data-qa="keypress">
```

Comment:
code is not aligned, please check other places 
```suggestion
        <input 
          class="form__input form__input--big"
          placeholder="Try “Los Angeles”"
          data-qa="keypress"
         >
```